script.js

```diff
@@ -1,4 +1,336 @@
+      }
+    }
+    // player2 turn
+    if (playerTurn == "Player 2") {
+      var diceRoll1 = randomDiceRoll();
+      var diceRoll2 = randomDiceRoll();
+      dice1 = diceRoll1; // storing player 2 numbers into global
+      dice2 = diceRoll2;
+      myOutputValue =
+        player2Name +
+        " has rolled " +
+        dice1 +
+        " and " +
+        dice2 +
+        ". <br> Select the sequence.";
+      sequenceToggle = "ON";
+      playerTurn = "Player 2 Sequence Toggle";
+      return myOutputValue;
+    }
+    // player 2 selection
+    if (sequenceToggle == "ON" && playerTurn == "Player 2 Sequence Toggle") {
+      if (input == "1") {
+        var sequence1 = selectSequence1(dice1, dice2);
+        player2Storage = sequence1;
+        player2 = player2 + player2Storage;
+        sequenceToggle = "OFF";
+        playerTurn = "Player 1";
+      }
+      if (input == "2") {
+        var sequence2 = selectSequence2(dice2, dice1);
+        player2Storage = sequence2;
+        player2 = player2 + player2Storage;
+        sequenceToggle = "OFF";
+        playerTurn = "Player 1";
+      }
+    }
+    // compare winners and message
+    var message = "";
+    if (player1 == player2) {
+      message = "Score is tied now";
+    }
+    if (player1 > player2) {
+      message = player1Name + " is leading!";
+    }
+    if (player2 > player1) {
+      message = player2Name + " is leading!";
+    }
+    return (
+      "Current Game Mode: " +
+      gameMode +
+      "<br>" +
+      message +
+      "<br>" +
+      player1Name +
+      " score now is " +
+      player1 +
+      "<br>" +
+      player2Name +
+      " score now is " +
+      player2 +
+      "<br> Next Round."
+    );
+  }
+  if (gameMode == "Lowest") {
+    if (input == "End") {
+      playerTurn = "End";
+      return (myOutputValue = "Whats your next game? Normal/Lowest/Auto");
+    }
+    // player1 turn
+    if (playerTurn == "Player 1") {
+      var diceRoll1 = randomDiceRoll();
+      var diceRoll2 = randomDiceRoll();
+      dice1 = diceRoll1; // storing player 1 numbers into global
+      dice2 = diceRoll2;
+      myOutputValue =
+        player1Name +
+        " has rolled " +
+        dice1 +
+        " and " +
+        dice2 +
+        ". <br> Select the sequence.";
+      sequenceToggle = "ON";
+      playerTurn = "Player 1 Sequence Toggle";
+      return myOutputValue;
+    }
+    // player 1 selection
+    if (sequenceToggle == "ON" && playerTurn == "Player 1 Sequence Toggle") {
+      if (input == "1") {
+        var sequence1 = selectSequence1(dice1, dice2);
+        player1Storage = sequence1;
+        player1 = player1 + player1Storage;
+        sequenceToggle = "OFF";
+        playerTurn = "Player 2";
+      }
+      if (input == "2") {
+        var sequence2 = selectSequence2(dice2, dice1);
+        player1Storage = sequence2;
+        player1 = player1 + player1Storage;
+        sequenceToggle = "OFF";
+        playerTurn = "Player 2";
+      }
+    }
```

Comment:
same as above , you are using the same logic for normal mode  as well , instead of highest win , in this case it's the lowest. You could make the normal mode into a function , within that function you could 

if (gameMode == "Lowest") {
if( message == ''win''){
// normal game win change to lowest game lose
}else if (message =="lose"){
//normal game lose change to lowest game win
}
}